README.md

```diff
@@ -90,115 +119,119 @@ require('orgmode').setup({
 })
 ```
 
-Or if you are using `init.vim`:
+Or if you are using `init.vim`, wrap the above snippet like so:
 ```vim
 " init.vim
 lua << EOF
 
--- Load custom tree-sitter grammar for org filetype
-require('orgmode').setup_ts_grammar()
+require('orgmode').setup_ts_grammar() ...
 
--- Tree-sitter configuration
-require'nvim-treesitter.configs'.setup {
-  -- If TS highlights are not enabled at all, or disabled via `disable` prop, highlighting will fallback to default Vim syntax highlighting
-  highlight = {
-    enable = true,
-    additional_vim_regex_highlighting = {'org'}, -- Required for spellcheck, some LaTex highlights and code block highlights that do not have ts grammar
-  },
-  ensure_installed = {'org'}, -- Or run :TSUpdate org
-}
-
-require('orgmode').setup({
-  org_agenda_files = {'~/Dropbox/org/*', '~/my-orgs/**/*'},
-  org_default_notes_file = '~/Dropbox/org/refile.org',
-})
 EOF
 ```
+#### Completion
 
-* **Open agenda prompt**: <kbd>\<Leader\>oa</kbd>
-* **Open capture prompt**: <kbd>\<Leader\>oc</kbd>
-* In any orgmode buffer press <kbd>g?</kbd> for help
-
-If you are new to Orgmode, see [Getting started](/DOCS.md#getting-started-with-orgmode) section in Docs.
-
-### Completion
-If you use [nvim-compe](https://github.com/hrsh7th/nvim-compe) and want
-to enable autocompletion, add this to your compe config:
+<details>
+  <summary><a href="https://github.com/hrsh7th/nvim-compe"><b>nvim-compe</b></a></summary>
+  </br>
 
 ```lua
-require'compe'.setup({
+require('compe').setup({
   source = {
     orgmode = true
   }
 })
 ```
 
-For [nvim-cmp](https://github.com/hrsh7th/nvim-cmp), add `orgmode` to list of sources:
+</details>
+
+<details>
+  <summary><a href="https://github.com/hrsh7th/nvim-cmp"><b>nvim-cmp</b></a></summary>
+  </br>
+
 ```lua
-require'cmp'.setup({
+require('cmp').setup({
   sources = {
     { name = 'orgmode' }
   }
 })
 ```
 
-For [completion.nvim](https://github.com/nvim-lua/completion-nvim), just add `omni` mode to chain complete list and add additional keyword chars:
+</details>
+
+<details>
+  <summary><a href="https://github.com/nvim-lua/completion-nvim"><b>completion-nvim</b></a></summary>
+  </br>
+
 ```lua
 vim.g.completion_chain_complete_list = {
   org = {
     { mode = 'omni'},
   },
 }
+-- add additional keyword chars
 vim.cmd[[autocmd FileType org setlocal iskeyword+=:,#,+]]
 ```
 
+</details>
+
 Or just use `omnifunc` via <kbd>\<C-x\>\<C-o\></kbd>
```

Comment:
@kristijanhusak I don't know if this line belongs to `completion-nvim`

Reply:
No, it should be separate paragraph. You can probably also put it in `details` tag.